Code/Framework/AzToolsFramework/AzToolsFramework/API/ToolsApplicationAPI.h

```diff
@@ -59,6 +61,17 @@ namespace AzToolsFramework
     //! Return true to accept this type of component.
     using ComponentFilter = AZStd::function<bool(const AZ::SerializeContext::ClassData&)>;
 
+    // when a property is modified, we attempt to retrieve the value that comes out in response to the Property Modification function that you may supply
+    // if you return anything other than Refresh_None, the tree may be queued for update:
+    enum PropertyModificationRefreshLevel : int
```

Comment:
You do know that you can forward declare an enum?
```c++
    enum PropertyModificationRefreshLevel : int;
```

Reply:
Yes, I wanted to move it as part of this change though to reduce the number of places that needed to include `PropertyEditorAPI.h`, which pulls in more includes than `ToolsApplicationAPI.h`